README.md

```diff
@@ -82,6 +82,14 @@ This plugin will look for linters in the [`ale_linters`](ale_linters) directory.
 Each directory within corresponds to a particular filetype in Vim, and each file
 in each directory corresponds to the name of a particular linter.
 
+### Always showing gutter
+To prevent left gutter to disappear when there is no errors set
```

Comment:
The grammar isn't the best here. You could write something like, "You can keep the sign gutter open at all times by setting the `g:ale_sign_column_always` to `1`."

Maybe we should add an "Options" section which explains all of the options, even though there's a link to the .vim file.